src/button/index.js

```diff
@@ -14,15 +14,16 @@ const styles = {
     textTransform: 'uppercase',
     background:'rgba(0,0,0,0)',
     fontSize: 14,
-    padding: [14, 54],
+    padding: [18, 54],
     borderRadius: 500,
     border: [3, 'solid', strokeColor],
     color:strokeColor,
     fontWeight: 700,
     fontFamily: body,
-    letterSpacing: 1.5,
+    letterSpacing: 2,
     lineHeight:1.5,
     cursor: 'pointer',
+    transitionDuration: '0.2s',
```

Comment:
This transition duration doesn't actually apply to anything, you'll need to at least define a property. See the [MDN article for `transition`](https://developer.mozilla.org/en-US/docs/Web/CSS/transition)